functions/mqpublish.go

```diff
@@ -321,7 +322,12 @@ func checkBroker(broker string) error {
 		return errors.New("nslookup failed for broker ... check dns records")
 	}
 	pinger := ping.NewTCPing()
-	intPort, err := strconv.Atoi(u.Port())
+	var intPort int
+	if strings.Contains(broker, "mqtt") {
+		intPort = 8083
+	} else {
+		intPort = 1883
```

Comment:
these ports won't be fixed, they can be anything

Reply:
the broker endpoint received from server is of the format `wss://broker.NETMAKER_BASE_DOMAIN/mqtt`, hence it wont contain the port, that's why we need to fix them

Reply:
@0xdcarns because it will be reverse proxied by caddy to the correct port based on `broker.NETMAKER_BASE_DOMAIN`

Reply:
removed `checkBroker` function